src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -153,7 +153,14 @@
           (reduce-kv (fn [acc k v]
                        (assoc acc k {:amount v :token-symbol token-symbol}))
                      {}
-                     network-amounts)]
+                     network-amounts)
+          network-values-sanitized (into {}
```

Comment:
should be in some utilities? or at least abstracted to a named fucnction in this file? 🤔

Reply:
@briansztamfater  it seems we want to preserve the keys and just update the values.

Consider using `update-vals` :+1: